components/tabs/src/sync/bridge.rs

```diff
@@ -372,6 +371,19 @@ mod tests {
                     "lastUsed": 1643764207
                 }]
             }),
+            // test an updated payload will replace the previous record
+            json!({
```

Comment:
Good catch -- I ended up moving this "test" to the engine since it was mainly making sure the apply_incoming works as intended and that's a better fit in the engine.rs